pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -301,14 +301,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
     versionCheckHook
   ];
   versionCheckProgram = "${placeholder "out"}/bin/zeditor";
-  versionCheckProgramArg = [ "--version" ];
+  versionCheckProgramArg = "--version";
   doInstallCheck = true;
 
   passthru = {
-    updateScript = gitUpdater {
-      rev-prefix = "v";
-      ignoredVersions = "(*-pre|0.999999.0|0.9999-temporary)";
-    };
+    updateScript = nix-update-script { };
```

Comment:
This still needs to ignore the previously specified versions, doesn't it?

Reply:
![grafik](https://github.com/user-attachments/assets/2c1162a7-4e40-4e20-a3a7-e4d79fea448d)

Unfortunately yes :')